packages/fxa-auth-server/lib/routes/subscriptions.ts

```diff
@@ -1027,6 +1027,10 @@ class DirectStripeRoutes {
     event: Stripe.Event
   ) {
     const invoice = event.data.object as Stripe.Invoice;
+    if (invoice.billing_reason === 'subscription_create') {
```

Comment:
Since the invoice was created because the subscription was being created, won't this be set the first time it fails? (Assuming I'm reading the docs correctly here: https://stripe.com/docs/api/invoices/object#invoice_object-billing_reason)

If I'm reading this right, repeat failures will then have billing_reason set to `subscription_update`.

Reply:
One more note, I *think* the only time we want to send a payment failed here is if the `billing_reason` is `subscription_cycle`, because it implies that it happened as a result of Stripe attempting to make the payment, vs. the initial or subsequent card updates.

Reply:
Giving this another read and yeah I think `subscription_cycle` is what we want here.